src/Button/native/Button.js

```diff
@@ -30,6 +48,15 @@ const styles = {
     }
     return props.theme.px(10);
   },
+  textColor(props) {
+    if (props.kind === 'filled') {
```

Comment:
`if (props.kind === 'filled' && !props.disabled)` ?

Reply:
disabled filled and enabled filled both will have white input as of now 🤔